content/en/docs/concepts/configuration/environment.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Environment Containers
```

Comment:
People will think that environment containers are a fourth kind alongside:
- sidecar containers
- init containers
- application containers

I would drop this page.